packages/material-ui/src/Collapse/Collapse.test.js

```diff
@@ -4,24 +4,19 @@ import { spy, stub, useFakeTimers } from 'sinon';
 import { createClientRender } from 'test/utils/createClientRender';
 import { getClasses } from '@material-ui/core/test-utils';
 import createMount from 'test/utils/createMount';
-import {
-  ThemeProvider,
-  createMuiTheme,
-  unstable_createMuiStrictModeTheme as createMuiStrictModeTheme,
-} from '@material-ui/core/styles';
+import { ThemeProvider, createMuiTheme } from '@material-ui/core/styles';
 import { Transition } from 'react-transition-group';
 import describeConformance from '../test-utils/describeConformance';
 import Collapse from './Collapse';
 
 describe('<Collapse />', () => {
-  // StrictModeViolation: uses react-transition-group
-  const mount = createMount({ strict: false });
+  const mount = createMount();
```

Comment:
While `createClientRender` is strict by default, `createMount` isn't (we wanted to avoid a breaking change)

```suggestion
  const mount = createMount({ strict: true });
```

Reply:
done